tests/tests.py

```diff
@@ -5,16 +5,16 @@
 import unittest
 import warnings
 
+sys.path.append(os.path.abspath("."))
+sys.path.append(os.path.abspath(".."))
```

Comment:
Why?

Reply:
TLDR; I need it to run the tests, because otherwise there is a failure to import the libkeepass library.

I can't be certain, but I believe that was added to enable running tests from either the root of the repository or the tests directory. According to the [docs](https://docs.python.org/3/using/cmdline.html#interface-options) on running a script file: `If the script name refers directly to a Python file, the directory containing that file is added to the start of sys.path, and the file is executed as the __main__ module.`  So the directory with the tests.py file is added to the path when run as `python tests/tests.py`, but we need the parent directory of tests in the path because that's where the libkeepass module is.  By adding `.` and `..` we allow the script to find the libkeepass module when started from either the root of the repository or the tests directory.  However, now I see that the script won't run from the tests directory anyway because the kdb files are found assuming the cwd is the root of the repository.  So perhaps adding `..` should be gotten rid of anyway.

Does running the tests work using the method described in the README.rst for you currently?

Reply:
I've always installed libkeepass with `pip install -e .` and run the tests from within `tests/`.

Reply:
I think this should be removed.